metrics/src/main/scala/com/mesosphere/usi/metrics/Metrics.scala

```diff
@@ -0,0 +1,76 @@
+package com.mesosphere.usi.metrics
```

Comment:
this is exactly what I didn't want...    why can we not have a `usi-commons` which has the common interfaces and models instead of having a individual project for everything we want to make extensible?

Reply:
Hehe, that is what I called `interfaces` but was overturned.

Reply:
Yep, that's the whole "interfaces" debate we had last week.